### DISCUSSION TITLE
Suggesting moving css-to-rn to it's own library

### DISCUSSION THREAD

**Discussion by divineniiquaye** [nativewind/nativewind#1140] [General]
I find the work done so far in the css-to-rn folder from the packages/react-native-css-interop very useful to the react native community.

Currently expo-router is looking into building direct support for tailwindcss, other developers like myself find the code in that folder very useful.

@marklawlor It will be great if you could possibly release it as an npm package, this will allow developers use your solution instead of copying your hard work code. Thanks

**Comment by danstepanov** [comment]
https://github.com/nativewind/react-native-css
